public/jobs.js

```diff
@@ -0,0 +1,91 @@
+import {
+  inputEnabled,
+  setDiv,
+  message,
+  setToken,
+  token,
+  enableInput,
+} from "./index.js";
+import { showLoginRegister } from "./loginRegister.js";
+import { showAddEdit } from "./addEdit.js";
+import { handleDelete } from "./delete.js";
+
+let jobsDiv = null;
+let jobsTable = null;
+let jobsTableHeader = null;
+
+export const handleJobs = () => {
+  jobsDiv = document.getElementById("jobs");
+  const logoff = document.getElementById("logoff");
+  const addJob = document.getElementById("add-job");
+  jobsTable = document.getElementById("jobs-table");
+  jobsTableHeader = document.getElementById("jobs-table-header");
+
+  jobsDiv.addEventListener("click", (e) => {
+    if (inputEnabled && e.target.nodeName === "BUTTON") {
+      if (e.target === addJob) {
+        showAddEdit(null);
+      } else if (e.target === logoff) {
+        setToken(null);
+
+        message.textContent = "You have been logged off.";
+
+        jobsTable.replaceChildren([jobsTableHeader]);
```

Comment:
Yes, good, this is important.  When the user logs off, the jobs table remains in browser memory, accessible via browser tools for perhaps a different user, unless you clear it in this way.